Packages/vcs/Lib/vcsvtk/fillareautils.py

```diff
@@ -33,8 +34,14 @@ def make_patterned_polydata(inputContours, fillareastyle=None,
     # Scaled the size to 2 times to make the pattern image of a finer resolution
     xBounds = bounds[1] - bounds[0]
     yBounds = bounds[3] - bounds[2]
-    xres = int(4.0*xBounds)
-    yres = int(4.0*yBounds)
+    if xBounds <= 1 and yBounds <= 1 and size is not None:
+        xBounds *= size[0]
```

Comment:
@sankhesh is kind of right you want your pattenr to be somewhat predictable, can we try to scale this pattern by the size of the window or something?

@chaosphere2112 with your changes would a fillarea pattern be different based on different values of the fa.worldcoordinate ?